portfolio/src/main/java/com/google/sps/servlets/ChartDataServlet.java

```diff
@@ -37,28 +39,15 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Load from datastore
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
-    PreparedQuery results = datastore.prepare(query);
-    Map wordsCount = new HashMap<String, Integer>();
-
-    // Go over all the comments
-    for (Entity entity : results.asIterable()) {
-      String content = (String) entity.getProperty("content");
+    CommentLoader cl = new CommentLoader();
+    List<Comment> comments = cl.getComments();
 
-      // Extract and clean the words from each comment
-      String[] words = content.split(" |!|\\.|\\,|\\?");
-      for (String word : words) {
-        word = word.trim().toLowerCase();
-        if (word.isEmpty()) {
-          continue;
-        }
-        int wordCount = wordsCount.containsKey(word) ? (int) wordsCount.get(word) : 0;
-        wordCount += 1;
-        wordsCount.put(word, wordCount);
-      }
-    }
+    Map<String, Long> wordsCount =
```

Comment:
Great usage!!

Usually the formatting would be something like:
```
array.stream()
  .filter(....)
  .map(...)
  .collect(...)

```

Reply:
Great, I was wondering about that! Thanks. Done.